requirements/bench.txt

```diff
@@ -29,3 +29,8 @@ keras>=2.4.3
 compiler_gym>=0.1.8 ; sys_platform == "linux"
 pymoo>=0.4.2.2
 bayes-optim>=0.2.5.5
+olymp==0.0.1b0 ; sys_platform == "linux"
+silence_tensorflow
+tensorflow_probability
+sqlalchemy
+seaborn
```

Comment:
are those really needed? looks like database and plotting tools so that feels unrelated

Reply:
Those are used internaly by Olympus. I could try removing them and see if there are no issues.

Reply:
It was Koncept512 that failed in pytests. Is it possible that it is related ?

Reply:
It works locally though.

Reply:
> It was Koncept512 that failed in pytests. Is it possible that it is related ?

Initially I would have expected it to be unrelated, but after a clean install of master on the CI, this issue does not appear. So what triggers the problem is necessaritly somewhere here, I would expect one of the packages, and because koncept512 is in tensorflow, probably silence_tensorflow or tensorflow_probability. tensorflow version and keras version are the same in master though so it's not "simply" a problem version :s
Is silence_tensorflow needed? Might be good to try without it I guess :s

Reply:
I will have to check all that.